botocore/__init__.py

```diff
@@ -44,10 +45,6 @@ def emit(self, record):
     ('DescribeStorediSCSIVolumes', '-'): 'describe-stored-iscsi-volumes',
     ('CreateStorediSCSIVolume', '_'): 'create_stored_iscsi_volume',
     ('CreateStorediSCSIVolume', '-'): 'create-stored-iscsi-volume',
```

Comment:
Just a side note. I happened to notice that our Ruby SDK uses a slightly different approach by [defining the special keywords "iSCSI" (and some others) only once](https://github.com/aws/aws-sdk-ruby/blob/d9e649c2b858647b68dcc80e38dff8f3ab1e752c/aws-sdk-core/lib/aws-sdk-core.rb#L4-L12), and then [preprocess them](https://github.com/aws/aws-sdk-ruby/blob/65fc6a6ea69792668763611af57c61db2c87a51d/aws-sdk-core/lib/seahorse/util.rb#L17-L20). Do we also need to handle "Ec2" and "ElastiCache" cases here?